bookmarks/images/tests/test_modelmixintestcase.py

```diff
@@ -0,0 +1,17 @@
+from django.test import TestCase
+from django.contrib.auth.models import User
+
+
+class ModelMixinTestCase(TestCase):
+    def setUp(self):
+        self.credentials = {
+            "username": "test",
+            "password": "123",
+        }
+        User.objects.create_user(**self.credentials)
+
+    def createImage(self):
+        self.client.login(**self.credentials)
+        self.client.get(
+            "https://127.0.0.1:8000/images/create/?title=test%20Django%20and%20Duke&url=https://upload.wikimedia.org/wikipedia/commons/8/85/Django_Reinhardt_and_Duke_Ellington_%28Gottlieb%29.jpg"
```

Comment:
Use reverse for the URL, it should not be hardcoded. Only the POST should create. GET should only retrieve the data.

Reply:
You can directly create image using Image.objects.create() here or use factory boy to create image. The created image should be returned